server/pkg/integrationapi/project.go

```diff
@@ -2,17 +2,69 @@ package integrationapi
 
 import (
 	"github.com/reearth/reearth-cms/server/pkg/project"
+	"github.com/reearth/reearthx/account/accountdomain/workspace"
 	"github.com/samber/lo"
 )
 
 func NewProject(p *project.Project) Project {
+	var publication *ProjectPublication = nil
+	if p.Publication() != nil {
+		publication = &ProjectPublication{
+			Scope:       ToProjectPublicationScope(p.Publication().Scope()),
+			AssetPublic: lo.ToPtr(p.Publication().AssetPublic()),
+			Token:       lo.ToPtr(p.Publication().Token()),
+		}
+	}
```

Comment:
if the publication is nil we should consider it as private, returning nil will confuse the user!